sale_invoice_delivery_state/models/partner.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    invoice_policy = fields.Selection(
+        [("partially", "Partially Delivered"), ("fully", "Fully Delivered")],
+        string="Invoice Policy",
```

Comment:
You can remove the string from the field definition, odoo by default will use the field name as label

Reply:
@gbrito , it's done.